test/unit/migrations/039-test.js

```diff
@@ -0,0 +1,312 @@
+const assert = require('assert')
+const migration39 = require('../../../app/scripts/migrations/039')
+
+describe('migration #39', () => {
+  it('should update the version metadata', (done) => {
+    const oldStorage = {
+      'meta': {
+        'version': 38,
+      },
+      'data': {},
+    }
+
+    migration39.migrate(oldStorage)
+      .then((newStorage) => {
+        assert.deepEqual(newStorage.meta, {
+          'version': 39,
+        })
+        done()
+      })
+      .catch(done)
```

Comment:
This test seems wrong. It will still pass if this catch block is triggered.

Reply:
It looks like the other tests are the same way. Any reason async/await isn't used instead? It makes it harder for this kinda mistake to happen.

Reply:
Why would it pass if this catch block is triggered? [The first argument to `done` is an error](https://mochajs.org/#asynchronous-code) and [the `catch` function provides the error](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/catch).

Reply:
Uh... right, you are correct. Nevermind. I was confused.